src/server/internalServer.cpp

```diff
@@ -462,18 +462,19 @@ bool InternalServer::start() {
   sockAddr6.sin6_port = htons(m_port);
 
   if (m_addr.empty()) {
-    if (0 != INADDR_ANY) {
+    if (0 != INADDR_ANY) { // what is the purpose of this? INADDR_ANY is always 0. Refer: http://www.castaglia.org/proftpd/doc/devel-guide/src/include/inet.h.html
```

Comment:
@veloman-yunkan should I also remove this entire if clause?

Reply:
I think that we should remove the `if` but not its body, thereby probably fixing a bug. To the best of my understanding, in this branch of the enclosing `if` `sockAddr6.sin6_addr = in6addr_any;` must be executed unconditionally. `sockAddr4.sin_addr.s_addr = htonl(INADDR_ANY)` is likely a no-op for `0 == INADDR_ANY`, but it doesn't hurt to have it in case there are systems where `INADDR_ANY` is non-zero.